core/domain/classifier_services.py

```diff
@@ -242,6 +242,51 @@ def get_classifier_training_job_from_model(classifier_training_job_model):
         classifier_training_job_model.data_schema_version)
 
 
+def _get_classifier_training_job_model_by_id(job_id, strict=True):
+    """Gets a classifier training job by a job_id.
+
+    Args:
+        job_id: str. ID of the classifier training job.
+        strict: bool. Raises Exception if ClassifierTrainingJobModel with
+            id not found and strict is set to True. Logs an error, otherwise.
+    
+    Returns:
+        classifier_training_job: ClassifierTrainingJobModel. NDB instance
+            of classifier training job.
+    
+    Raises:
+        Exception: Entity for class ClassifierTrainingJobModel with id not
+            found.
+    """
+    classifier_training_job_model = (
+        classifier_models.ClassifierTrainingJobModel.get(job_id, strict=strict))
+    if classifier_training_job_model:
+        classifier_training_job_model.classifier_data = (
```

Comment:
Ah, no, please don't modify storage models midstream. Instead, in general, convert to domain objects straightaway and try not to expose functions like this, even privately within the module. The conversion to domain objects should happen immediately after fetching -- in general, this should be done by a _get_classifier_training_job_from_model(job_model) function which returns a domain object with the classifier_data populated (by referencing the job ID in the filesystem).

Reply:
Done.